sdk/storage/Azure.Storage.Blobs/src/Generated/Models/QueryFormat.cs

```diff
@@ -23,5 +23,7 @@ public QueryFormat()
         public JsonTextConfigurationInternal JsonTextConfiguration { get; set; }
         /// <summary> arrow configuration. </summary>
         public ArrowTextConfigurationInternal ArrowConfiguration { get; set; }
+        /// <summary> Any object. </summary>
+        public object ParquetTextConfiguration { get; set; }
```

Comment:
how did this end up `object` ?

Reply:
The generator doesn't like it that `ParquetTextConfiguration` doesn't have any properties.  It is actually ok, `ParquetTextConfiguration` is optional if you specify `Parquet` for `QueryFormatType`.

If/when `ParquetTextConfiguration` gets additional properties, the generator will generate it correctly.

As long as this doesn't cause issues on the other platforms, this is a non-issue.